docs/blog/2019-11-11-gazette-october/index.mdx

```diff
@@ -0,0 +1,123 @@
+- [Improvements to how gatsby-plugin-offline handles custom link tags](https://github.com/gatsbyjs/gatsby/pull/16691) with the `rel=preconnect` attribute
+- A [fix for the `wrapPageElement` API while running `gatsby develop`](https://github.com/gatsbyjs/gatsby/pull/17111)
+- [Performance improvements to querying](https://github.com/gatsbyjs/gatsby/pull/17682), adding a nice speed boost when building larger sites.
+- By the same author [this PR improved Gatsby's type inference](https://github.com/gatsbyjs/gatsby/pull/18008), making it faster and less memory intensive.
+- A [fix for `gatsby-plugin-manifest`](https://github.com/gatsbyjs/gatsby/pull/17426) that ensures manifest links are updated after page navigation.
+- An issue that happened when [creating a resolver using the `context.nodeModel` API was fixed](https://github.com/gatsbyjs/gatsby/pull/18650).
+- [This PR](https://github.com/gatsbyjs/gatsby/pull/16686) reduces the number of files that are changed with each build. This should result in faster deploys on services like Netlify, particularly for larger sites.
+-
+
+* https://github.com/gatsbyjs/gatsby/pull/18533 fix(gatsby-cli): properly update chalk color level
+* https://github.com/gatsbyjs/gatsby/pull/18478 fix(gatsby): Fix matchpath ordering
+* fix(gatsby-plugin-netlify): don't add `/undefined` in generated `_headers` file #18925
+* https://github.com/gatsbyjs/gatsby/pull/18931 chore(docs): explain shadowing a bit more
+* fix(gatsby-cli): handle git commit failures https://github.com/gatsbyjs/gatsby/pull/18839
+* https://github.com/gatsbyjs/gatsby/pull/18916 chore(gatsby-image): Add missing type definitions for GatsbyImageProps
+
+## Site of the Month
+
+The latest site of the month is [IBM's Carbon Design System](https://www.carbondesignsystem.com/). What's it all about?
+
+> Carbon is IBM’s open-source design system for products and experiences. With the IBM Design Language as its foundation, the system consists of working code, design tools and resources, human interface guidelines, and a vibrant community of contributors.
+
+Seems like everyone is building design systems these days - for good reason. There's a lot to like in here - based on guidelines to be open, inclusive, modular, user-based and consistent. Components are availble for vanilla JS, React, Angular, and Vue.
+
+- Design system with lots of well thought out components and clear guidelines
+- Open source
+- Scale!
+
+## Theme of the Month
+
+![screenshot of gatsby-theme-firebase's readme](./gatsby-theme-firebase.png)
+
+[`gatsby-theme-firebase`](https://github.com/epilande/gatsby-theme-firebase) might be the quickest way to add full-featured auth to your site, and it's our theme of the month for this month!
+
+Add the theme, your Firebase credentials and off you go. Want to change anything? Use shadowing to override whichever components you want.
+
+## How to Equip your Marketing Team for Success
```

Comment:
We probably want to update this 😄 as it's outdated

Reply:
Yeah, I’m getting this in today. Will update this with a section about Gatsby Cloud